tests/auth/test_login.py

```diff
@@ -25,15 +23,11 @@ def test_unactivated_user_login(flask_client):
 
 
 def test_activated_user_login(flask_client):
-    """Start with a blank database."""
-
-    # create user, user is activated
-    User.create(email="a@b.c", password="password", name="Test User", activated=True)
-    Session.commit()
+    user = create_new_user()
 
     r = flask_client.post(
         url_for("auth.login"),
-        data={"email": "a@b.c", "password": "password"},
+        data={"email": user.email, "password": "password"},
         follow_redirects=True,
     )
```

Comment:
`login(flask_client)` could be used directly here, couldn't it?

Reply:
yeah that's true, this test is removed in https://github.com/simple-login/app/pull/899/commits/b545ebaeb117e95ed5f16b497c16cb82a8a35414